src/components/new-request/search.vue

```diff
@@ -178,6 +178,28 @@
       </v-col>
     </v-row>
 
+    <!-- business lookup -->
+    <v-row no-gutters>
+      <v-col cols="12">
+        <template>
+          <!-- Search for business identifier or name -->
+          <BusinessLookup
+            @business="businessName = $event.name; businessIdentifier = $event.identifier"
+          />
+
+          <template v-if="businessIdentifier">
```

Comment:
Note to other devs working on Name Change flow, Amalgamation flow, Alteration flow or Reactivate flow -- delete this section and use the business name/ identifier/ legal type from the business search.